content/800-data-platform/10-team-members/050-user-roles.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: 'User roles'
+metaTitle: 'User roles'
+metaDescription: 'The Prisma Data Platform supports the Admin, Developer, Collaborator, and Viewer roles that you can assign to your team members and control which parts of the project they can only view and also edit.'
+toc: true
+---
+
+<TopBlock>
+
+These roles are currently supported in the Prisma Data Platform:
+
+- **Admin**
+- **Developer**
+- **Collaborator**
+- **Viewer**
+
+To add users, you need to use their GitHub username and they need to have an account in the Platform.
+
+</TopBlock>
+
+## Roles and permissions
+
+| Feature | Project owner | Admin | Developer | Collaborator | Viewer |
+| -------------- | :---- | :-------- | :----------- | :----- | :----- |
+| **Data Browser** |  |  |  |  |  |
+| View data | ✅ | ✅ | ✅ | ✅ | ✅ | 
+| Edit data | ✅ | ✅ | ✅ | ✅ | ⛔️ |
+| **Query Console** |  |  |  |  |  |
+| Run default queries | ✅ | ✅ | ✅ | ✅ | ✅ |
+| Add new queries | ✅ | ✅ | ✅ | ✅ | ✅ |
```

Comment:
No, I had an incorrect assumption that we might be preventing users with specific roles to write their own queries in case of us not willing to allow them to overload the database or something to that end.

I've now merged "run and add" into a single row.

Reply:
"Add" indicated that they would still be there for other users that also can access the account.
I am not sure if that is the case, or they are only stored for myself or even not at all.

Reply:
Wow, so we completely came thinking about this from very different directions.

I added a query as a 'Viewer' and its stored there for my personal later usage.

I added another one as the 'Project owner' and it's the same. Neither can see what the other added.

Reply:
It still makes sense to me that 'Add and run queries' is available for all user roles.

But you could have a different view on this so I'd be glad to hear.